rwandaemr-imb-butaro/openmrs-distro.properties

```diff
@@ -23,20 +35,23 @@ omod.omodexport=${omodExportVersion}
 omod.orderextension=${orderextensionVersion}
 #omod.patientsummary=${patientsummaryVersion}
 omod.pharmacymanagement=${pharmacymanagementVersion}
+omod.providermanagement=${providermanagementVersion}
 omod.reportingcompatibility=${reportingcompatibilityVersion}
 omod.reportingobjectgroup=${reportingobjectgroupVersion}
 omod.reporting=${reportingVersion}
+omod.reportingrest=${reportingrestVersion}
+omod.reportingui=${reportinguiVersion}
 omod.rowperpatientreports=${rowperpatientreportsVersion}
 omod.rwandahivflowsheet=${rwandahivflowsheetVersion}
 omod.rwandaprimarycare=${rwandaprimarycareVersion}
 omod.rwandareports=${rwandareportsVersion}
 omod.serialization.xstream=${serializationVersion}
 omod.serialization.xstream.type=omod
+omod.spa=${spaVersion}
 omod.sync=${syncVersion}
+omod.uicommons=${uicommonsVersion}
+omod.uiframework=${uiframeworkVersion}
+omod.uilibrary=${uilibraryVersion}
 omod.usagestatistics=${usagestatisticsVersion}
 omod.webservices.rest=${webservicesRestVersion}
-omod.spa=${spamodulesVersion}
-omod.reportingrest = ${reportingrestVersion}
-omod.imbpathologyrequest=${imbpathologyrequestVersion}
-omod.fhir2=${fhir2Version}
```

Comment:
These aren't actually removed, I just like to keep everything organized alphabetically to more easily look up what is present.  If we can try to maintain that, it would be good.